Python/Product/VSInterpreters/RegistryWatcher.cs

```diff
@@ -485,24 +486,24 @@ enum RegNotifyChange : uint {
 
         private static RegistryHive ParseRegistryKey(string key, out string subkey) {
             int firstPart = key.IndexOf('\\');
-            if (firstPart < 0 || !key.StartsWith("HKEY", StringComparison.InvariantCultureIgnoreCase)) {
+            if (firstPart < 0 || !key.StartsWithOrdinal("HKEY", ignoreCase: true)) {
                 throw new ArgumentException("Invalid registry key: " + key, "key");
             }
             var hive = key.Remove(firstPart);
             subkey = key.Substring(firstPart + 1);
-            if (hive.Equals("HKEY_CURRENT_USER", StringComparison.InvariantCultureIgnoreCase)) {
+            if (hive.Equals("HKEY_CURRENT_USER", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
`EqualsOrdinalIgnoreCase`

Reply:
Don't see the benefit here, there aren't so many places where we do this.